specification/batch/resource-manager/Microsoft.Batch/stable/2021-01-01/BatchManagement.json

```diff
@@ -3543,6 +3584,55 @@
       },
       "description": "Pool properties."
     },
+    "BatchPoolIdentity": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the Batch Pool.",
+          "enum": [
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "PoolIdentityType",
+            "modelAsString": false,
```

Comment:
Suggest to use:  "modelAsString": true,  if so, it will be easy for you add more enum value like systemAssigned in the future without breaking change for SDK

Reply:
These values aren't modeled as strings in other Azure APIs that support managed identities (such as VMs or VMSS). It seems like it would be better to be consistent here.